tiingo/api.py

```diff
@@ -310,8 +310,9 @@ def get_dataframe(
                 """When tickers is provided as a list, metric_name is a required argument.
             Please provide a metric_name, or call this method with one ticker at a time."""
             )
-
         params = {"format": fmt, "resampleFreq": frequency}
+        if includeIntradayVolume:
```

Comment:
I don't think this will work, since `includeIntradayVolume` isn't defined. Once that is added, could you include a unit test that will cover this line?

Reply:
I added `includeIntradayVolume` as it was referenced in [#462](https://github.com/hydrosquall/tiingo-python/issues/462) which I figured `includeIntradayVolume` had already been defined.

Do you have any recommendations for the location where should I add it?

If you can give me a few pointers on the location of the definition, I can probably work up a unittest PR for `includeIntradayVolume` easily enough.

Reply:
Hey @datatalking , thanks for the patience

What I meant was that the current code doesn't work because this variable was never defined.

I think you would probably want to add `includeIntradayVolume` as an additional parameter to the `get_dataframe` method . 

https://github.com/hydrosquall/tiingo-python/blob/5142324576cac0537aba1b4be2d3d4bb6e0f31c5/tiingo/api.py#L265-L267